conf/app.routes

```diff
@@ -1,8 +1,8 @@
 # microservice specific routes
 
-GET /eligibility-check       @uk.gov.hmrc.helptosave.controllers.EligibilityCheckController.eligibilityCheck(nino: String)
-GET /enrolment-status        @uk.gov.hmrc.helptosave.controllers.EnrolmentStoreController.getEnrolmentStatus(nino: String)
-GET /enrol-user              @uk.gov.hmrc.helptosave.controllers.EnrolmentStoreController.enrol(nino: String)
-GET /set-itmp-flag           @uk.gov.hmrc.helptosave.controllers.EnrolmentStoreController.setITMPFlag(nino: String)
-GET /store-email             @uk.gov.hmrc.helptosave.controllers.EmailStoreController.store(email: String, nino: String)
-GET /get-email               @uk.gov.hmrc.helptosave.controllers.EmailStoreController.get(nino: String)
+GET /eligibility-check       @uk.gov.hmrc.helptosave.controllers.EligibilityCheckController.eligibilityCheck()
```

Comment:
putting this change here now means that we need to update the frontend at the same time otherwise the master branches of FE and BE will not be able to communicate with each other

Reply:
yes, getting the frontend ready

Reply:
https://github.com/hmrc/help-to-save-frontend/pull/219